flex/views/experimentplatform/ExperimentPlatformWindow.mxml

```diff
@@ -191,10 +191,10 @@
 
 			// called from onSaveExperimentPlatform here and in ExperimentPlatformAddWindow
 			public function afterSuccessfulSave(savedCodeRequestCategory:String):void {
-				this.enabled = false;
 				codeRequestCategorySelected = savedCodeRequestCategory;
 				// Must reload dictionary manager in case user changed seqlibprotocol.  Need to get platform list
 				// after reload complete so various dropdowns are ok.
+				this.refreshPlatformList();
 				parentApplication.dictionaryManager.reload();
```

Comment:
The comment just above your new line says that the platform list should be refreshed after the dictionary manager is reloaded, but you are doing it before the dictionary reload. Is that okay? If so, you should probably change the comment so it's less confusing.